ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js

```diff
@@ -45,16 +44,15 @@ const TableWidgetFixableCves = ({
     const [cveSort, setCveSort] = useState(defaultCveSort);
 
     const { isFeatureFlagEnabled } = useFeatureFlags();
-    const showVMUpdates = isFeatureFlagEnabled('ROX_POSTGRES_DATASTORE');
```

Comment:
I think we can also remove `useFeatureFlags()` completely from this file too ?

Reply:
Good question. It is still used indirectly:

1. `getCveTableColumns(workflowState, isFeatureFlagEnabled)`
2. `getFilteredCVEColumns(columns, workflowState, isFeatureFlagEnabled)`
3. `isFeatureFlagEnabled('ROX_ACTIVE_VULN_MGMT')`